app/views/user_mailer/plan_shopping_completed.html.erb

```diff
@@ -224,7 +224,7 @@
                                       8:00am -
                                       8:00pm<br />
                                       E-Mail:
-                                      <%= Settings.contant_center.email_address %></span><br />
+                                      <%= Settings.contact_center.email_address %></span><br />
```

Comment:
this should come from the Resource Registry, shouldn't it?

Reply:
Well, yes. But this was a typo which led me to think it was either never being rendered to begin with, or something of that nature. And I wasn't going to swap out everything with resource registry and then turn around and change the entire file anyway for a new client until I got requirements.